src/App.js

```diff
@@ -1,26 +1,700 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react';
+
 import './App.css';
+import Table from './table/Table';
+import Navigation from './table/Navigation';
+
+class App extends Component {
+  state = {
+    showTable: false,
+
+    showApp: 'Void',
+    showTests: 'Void',
+    showEnv: 'Void',
+    testCount: 14,
+
+    testResults: [
```

Comment:
Can we put this in a 'constants' directory - same thing we do in Cypress tests for test data.